rules.yaml

```diff
@@ -275,6 +275,7 @@
 - { name: libfaketime,                 verpat: ".*201[67].*",                                ignorever: true } # FreeBSD is using git snapshot
 - { name: kbuild,                      verlonger: 3,                                         ignorever: true } # 0.1.9998.8.2814.25 @ pkgsrc
 - { name: kbuild,                      verpat: ".*[rs].*",                                   ignorever: true } # 0.1.9998.r2813 | 0.1.9998svn2814
+- { name: ksh,                         ver: "20120801",                  family: [ fedora ], ignorever: true }
```

Comment:
Let's postpone this. More repos not available on repology.org yet contain weird versions for ksh, so this will need more consideration.